ponder.config.ts

```diff
@@ -1,32 +1,32 @@
 import { deepMergeRecursive } from "./src/lib/helpers";
-import { type IntersectionOf, getActivePlugins } from "./src/lib/plugin-helpers";
+import { type MergedTypes, getActivePlugins } from "./src/lib/plugin-helpers";
 import * as baseEthPlugin from "./src/plugins/base.eth/ponder.config";
 import * as ethPlugin from "./src/plugins/eth/ponder.config";
 import * as lineaEthPlugin from "./src/plugins/linea.eth/ponder.config";
 
 // list of all available plugins
-// any of them can be activated in the runtime
-const plugins = [baseEthPlugin, ethPlugin, lineaEthPlugin] as const;
+// any available plugin can be activated at runtime
+const availablePlugins = [baseEthPlugin, ethPlugin, lineaEthPlugin] as const;
 
-// intersection of all available plugin configs to support correct typechecking
+// merge of all available plugin configs to support correct typechecking
 // of the indexing handlers
-type AllPluginConfigs = IntersectionOf<(typeof plugins)[number]["config"]>;
+type AllPluginConfigs = MergedTypes<(typeof availablePlugins)[number]["config"]>;
 
 // Activates the indexing handlers of activated plugins and
 // returns the intersection of their combined config.
```

Comment:
```suggestion
// Statically typed as the merge of all available plugin configs. However at runtime returns the merge of all activated plugin configs.
```

I'm not sure if I'm understanding this correctly. Is what I wrote above correct? Appreciate if you could refine if needed :+1: